stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/content/MessageAttachmentsContent.kt

```diff
@@ -39,6 +39,7 @@ import io.getstream.chat.android.uiutils.extension.hasLink
 public fun MessageAttachmentsContent(
     message: Message,
     onLongItemClick: (Message) -> Unit,
+    modifier: Modifier = Modifier,
```

Comment:
This is a breaking change and if we decide to keep it, deprecation + new essentially copied function would be the norm.
Without this our aspect ratio minded gifs are not always centered when you use them inside the ephemeral message window and they look awkward.

Please run the app from this branch and check if it looks fine to you.
Also change the scaling factor down to something like `1f` or `2f` and check if that is fine as well.